modules/osd-create-cluster-ccs-aws.adoc

```diff
@@ -0,0 +1,227 @@
+// Module included in the following assemblies:
+//
+// * osd_install_access_delete_cluster/creating-an-aws-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="osd-create-aws-cluster-ccs_{context}"]
+= Creating a cluster on AWS
+
+By using the Customer Cloud Subscription (CCS) billing model, you can create an {product-title} cluster in an existing
+{AWS} account that you own.
+
+You can also select the Red Hat cloud account infrastructure type to deploy {product-title} in a cloud provider account that is owned by Red Hat.
+
+Complete the following prerequisites to use the CCS model to deploy and manage {product-title} into your AWS account.
+
+.Prerequisites
+
+* You have configured your AWS account for use with {product-title}.
+* You have not deployed any services in your AWS account.
+* You have configured the AWS account quotas and limits that are required to support the desired cluster size.
+* You have an `osdCcsAdmin` AWS Identity and Access Management (IAM) user with the `AdministratorAccess` policy attached.
+* You have set up a service control policy (SCP) in your AWS organization. For more information, see _Minimum required service control policy (SCP)_.
+* Consider having *Business Support* or higher from AWS.
+* If you are configuring a cluster-wide proxy, you have verified that the proxy is accessible from the VPC that the cluster is being installed into. The proxy must also be accessible from the private subnets of the VPC.
+
+.Procedure
+
+. Log in to {cluster-manager-url} and click *Create cluster*.
+
+. On the *Create an OpenShift cluster* page, select *Create cluster* in the *Red Hat OpenShift Dedicated* row.
```

Comment:
The default route of cluster manager url will be the overview page. i.e. https://console.redhat.com/openshift/ would redirect to https://console.redhat.com/openshift/overview. In overview page , you have inline options
![Screenshot 2024-12-18 at 10 55 45 AM](https://github.com/user-attachments/assets/4f869942-80dc-4419-8d2d-d6a507225d96)


Hence It would be good to rearrange both step 1 and 2 accordingly.
Suggestions would be
1. Log in to {cluster-manager-url}
2. On the *Overview* page, select  *Create cluster* in the *Red Hat OpenShift Dedicated* card.